lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-xs 6px $nx-spacing-md;
+  max-width: 824px;
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+  padding-right: $nx-spacing-xs;
+}
+
+.nx-tag--selectable {
+  cursor: pointer;
+  display: inline-flex;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+    box-shadow: $nx-focus-box-shadow;
+    outline: 0;
+  }
+
+  &:hover {
+    border-color: $nx-grey-900;
+  }
+}
+
+.nx-tag--selected {
+  background-color: $nx-blue-800;
+  border-color: $nx-blue-800;
+  color: $nx-text-color-dark;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+  }
+
+  .nx-tag__action {
+    .nx-icon {
+      transform: rotate(-45deg);
```

Comment:
Would probably be a little bit more semantic to use `faTimesCircle` rather than rotating the icon.

Reply:
> Would probably be a little bit more semantic to use `faTimesCircle` rather than rotating the icon.

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/97b74ac924814a28e8e00a14083658738bd39933